.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,38 @@
-## Description
-
-Describe your changes in detail.
-
-## Motivation and Context
-
-Why is this change required? What problem does it solve?
-If it fixes an open issue, please link to the issue here.
-You can use the syntax `close #15213` if this solves the issue #15213
```

Comment:
Maybe it's better to keep "Related issue".

Reply:
Added